settings.gradle

```diff
@@ -55,3 +54,40 @@ if (properties.getOrDefault('wp.BUILD_GUTENBERG_FROM_SOURCE', false).toBoolean()
     project(':react-native-reanimated').projectDir = new File(rootProject.projectDir, 'libs/gutenberg-mobile/gutenberg/node_modules/react-native-reanimated/android')
 }
 
+gradle.ext.fluxCBinaryPath = "com.github.wordpress-mobile.WordPress-FluxC-Android"
+gradle.ext.wputilsBinaryPath = "org.wordpress:utils"
+
+def localBuilds = new File('local-builds.gradle')
+if (localBuilds.exists()) {
+    apply from: localBuilds
+
+    /*
+    In order to add a new included build:
+    1. Define a property for its path in the `local-builds.gradle-example` file. 
+    2. Check if that property exists in `ext` since it may be commented out.
+    3. Include the build using the property.
+    4. Add dependencySubstitution block and substitute the binary module with the project.
+
+    Note that `project` dependencies are resolved in the context of the included build.
+    https://docs.gradle.org/current/javadoc/org/gradle/api/initialization/ConfigurableIncludedBuild.html#dependencySubstitution-org.gradle.api.Action-
+    */
+
+    if (ext.has("localFluxCPath")) {
+        includeBuild(ext.localFluxCPath) {
+            dependencySubstitution {
+                println "Substituting fluxc with the local build"
+                substitute module("$gradle.ext.fluxCBinaryPath:fluxc") with project(':fluxc')
+            }
+        }
+    }
+
+    if (ext.has("localWPUtilsPath")) {
+        includeBuild(ext.localWPUtilsPath) {
+            dependencySubstitution {
+                println "Substituting wputils with the local build"
+                substitute module("$gradle.ext.wputilsBinaryPath") with project(':WordPressUtils')
```

Comment:
I'd like to know why the substitution for FluxC has the `:fluxc` prefix while this doesn't, but I guess I'll have to figure it out by myself 😅